integration/spark/src/test/scala/org/apache/carbondata/view/rewrite/TestAllOperationsOnMV.scala

```diff
@@ -79,6 +90,12 @@ class TestAllOperationsOnMV extends QueryTest with BeforeAndAfterEach {
     }.getMessage.contains("Cannot add columns in a materialized view table default.dm1")
   }
 
+  test("test alter add complex column on MV table") {
+    intercept[ProcessMetaDataException] {
```

Comment:
This test case already exists. Please remove duplicate test cases

Reply:
done